### ISSUE TITLE
Bug: Tracim display error message (about TLM)

### ISSUE THREAD

**Issue by buxx** [tracim/tracim#3485]
## Description and expectations

Tracim display the following error message:

> Erreur
> Tracim rencontre un souci de connexion, merci de patienter ou de redémarrer votre navigateur si le problème persiste

### How to reproduce

1. I have a Tracim tab (pinned) in my Firefox Window.
1. I have a second Tracim tab with online edition active ([this](https://algoo.tracim.fr/ui/online_edition/workspaces/30/contents/14841))
1. After do F5 on my second tab, Tracim display the error

### Version information

- Tracim version: v3.0.4
- Web browser: Firefox
- Version: 80.0 (64 bits)
- Device / System: Kubuntu 20.04, in VM, personal computer

Console logs: 
[console-export-2020-9-2_12-19-59.txt](https://github.com/tracim/tracim/files/5161651/console-export-2020-9-2_12-19-59.txt)

**Comment by raphj** [comment]
Since we recently modified something related to the TLM connection in Tracim 3.1.1 (#3558), I suggest to wait and see if it still happens with that fix.

**Comment by buxx** [comment]
Today, after re-login and clicked on notification link:

![Sélection_048](https://user-images.githubusercontent.com/1104637/93178006-ed02d500-f733-11ea-958f-e10250e7f02d.png)

[console-export-2020-9-15_9-11-0.txt](https://github.com/tracim/tracim/files/5223058/console-export-2020-9-15_9-11-0.txt)

**Comment by raphj** [comment]
Seems related to #3524.

**Comment by raphj** [comment]
First, looking at the log, you provided the log of a tab that is a leader. This part seems relevant in the logs:

```
12:05:39,978 .:. TLM KeepAlive: tracim.app.js:73:747825
12:05:41,722 .:. TLM received:  
[...]
```

All is good, then a keep alive is received:

```
12:06:11,975 .:. TLM KeepAlive: 4 tracim.app.js:73:747825
```

Then, an error happens, almost 2 minutes later, without any event in-between:
```
12:08:02,518 TLM STATUS:  error tracim.app.js:73:750424
```

12:08:02,518 GLOBAL_dispatchEvent TracimLiveMessageStatusChanged 
Object { status: "error" }
tracim.app.js:73:456711
12:08:02,519 <Tracim> Custom event TracimLiveMessageStatusChanged 
```

Then an app is opened:
```
12:08:06,978 GLOBAL_renderAppFullscreen  [...]
12:08:06,978 GLOBAL_dispatchEvent collaborative_document_edition_showApp  [...]
```

Then a keepAlive is received, and a TLM more than a minute later:

```
12:08:09,225 .:. TLM KeepAlive: 3 tracim.app.js:73:747825
12:09:32,396 .:. TLM received:  
```

The TLM status **IS NOT UPDATED** here. Hence the message I think.

I also note that your connection to the TLM seems very flaky, looking at the log. There are 6 occurrences of "TLM STATUS:  error".

We need to understand why the status is not updated. We may need to set the status to OK when we receive a KeepAlive or a TLM too (which would be a good workaround in this case).

**Comment by raphj** [comment]
It could be fixed with #3606. Let's test this fix in preprod to see.
